phpBB/phpbb/install/helper/navigation/navigation_provider.php

```diff
@@ -0,0 +1,115 @@
+<?php
```

Comment:
Fixed in places where it was trivial, other stuff can be fixed latter if someone wants to